### PR TITLE
feat: add Raspberry Pi System overview dashboard

### PR DESCRIPTION
**Pull request by ivankudibal**

### CODE REVIEW

File: raspberry-pi/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+influx apply -u https://raw.githubusercontent.com/influxdata/community-templates/master/raspberry-pi/raspberry-pi-system.yml
+```
+
+## Included Resources
+
+- 1 Bucket: `telegraf`, 7d retention
+- Labels: `raspberry-pi` + Telegraf Plugin Labels
+- 1 Telegraf Configuration
+- 1 Dashboard: `Raspberry Pi System`
+- 2 Variables: `bucket` and `linux_host`
```
Comment (mhall119) [inline raspberry-pi/README.md:29]: These variables are not defined in the template

File: raspberry-pi/raspberry-pi-system.yml [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+          ## If set to true, do no set the "host" tag in the telegraf agent.
+          omit_hostname = false
+        [[outputs.influxdb_v2]]	
+          urls = ["$INFLUX_HOST"]
+          ## Token for authentication.
+          token = "$INFLUX_TOKEN"
+          ## Organization is the name of the organization you wish to write to; must exist.
+          organization = "$INFLUX_ORG"
+          ## Destination bucket to write into.
+          bucket = "telegraf"
```
Comment (mhall119) [inline raspberry-pi/raspberry-pi-system.yml:462]: Your Flux queries use the `v.bucket` variable, but your Telegraf config is hard-coded to use `telegraf` for the bucket. If you want bucket to be user-selectable the Telegraf config should use an environment variable such as $INFLUX_BUCKET here

**Review by mhall119 — CHANGES_REQUESTED** [review]
A couple of changes requested.

Also can you `git mv raspberry-pi/README.md raspberry-pi/readme.md`? We are working to standardize on this naming convention to make integration in the GUI easier.

**Comment by mhall119** [comment]
@ivankudibal this PR still has the old README.md file in it, can you remove it? Otherwise it looks good

**Comment by ivankudibal** [comment]
All changes were made, can you review and merge%20the%20template,%20pls?

> A couple of changes requested.
> 
> Also can you `git mv raspberry-pi/README.md raspberry-pi/readme.md`? We are working to standardize on this naming convention to make integration in the GUI easier.

done

**Review by mhall119 — APPROVED** [review]

**Pull request merged**